src/main/java/com/spanish_inquisition/battleship/server/MessageBus.java

```diff
@@ -66,4 +64,4 @@ public Message getMessageFromSender(int senderId) {
         messageBus.removeAll(newList);
         return msg;
     }
-}
```

Comment:
After we introduce packages this "package" modifiers will return to "public". But for now they can be as they are for better encapsulation.